coderd/coderdtest/coderdtest.go

```diff
@@ -787,6 +806,14 @@ func UpdateActiveTemplateVersion(t testing.TB, client *codersdk.Client, template
 	require.NoError(t, err)
 }
 
+// UpdateTemplateMeta updates the template meta for the given template.
+func UpdateTemplateMeta(t testing.TB, client *codersdk.Client, templateID uuid.UUID, meta codersdk.UpdateTemplateMeta) codersdk.Template {
```

Comment:
With a helper like this, would it make sense to ensure that all methods of `codersdk.Client` are implemented here instead of just select ones? This essentially turns 2 lines into 1 which arguably doesn't add a whole lot of value if you still don't know when to use `client` directly, and when to use `coderdtest`.

Reply:
The linter won't complain about passing the client into a `coderdtest` method, which I view as both a feature and a semantic difference:
- `coderdtest.X`: this thing is part of test setup.
- `client.X`: this is the thing we are actually testing where ensuring it works with the least-privileged RBAC role is crucial.

I don't think a 1:1 mapping of client methods -> coderdtest methods is warranted, especially as we have some `coderdtest` methods that call multiple client methods.